src/api/bucket_api.js

```diff
@@ -1522,5 +1528,19 @@ module.exports = {
                 }
             }
         },
+
+        log_based_replication_info: {
```

Comment:
maybe move `log_based_replication_info` into `replication policy` definition? they seem to always go together. 

Also, I think that `log_based_replication_info` is a bit of a mouthful. maybe rename it to `log_replication_info` or something?